src/components/Subscriptions/SubscriptionsCausesCancellationFlow.vue

```diff
@@ -8,6 +8,18 @@
 		<div class="causes-cancel-lightbox__content">
 			<label>
 				<textarea
+					class="
```

Comment:
Nice, think we should make a KvTextArea to keep these consistent?

Reply:
Yeah that would be nice, it would probably be a really simple component